### PR TITLE
Reorganizing Files and fixing class

### PR DESCRIPTION
**Pull request by jean-farinea**
1.Organizing files with 1 task per brench
2.Fixing Class to return a boolean value instead of String.

### CODE REVIEW

File: src/test/java/lesson2/task5/IsEvenOrOddTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package lesson2.task5;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class IsEvenOrOddTest {
+
+    @Test
+    void verifyIfNumberIsEvenOrOdd() {
```
Comment (strzeleckik) [inline src/test/java/lesson2/task5/IsEvenOrOddTest.java]: It's better to split it into two separate test functions like: shouldVerifyNumberAsEven, shouldVerifyNumberAsNotEven :)

Comment (strzeleckik) [reply src/test/java/lesson2/task5/IsEvenOrOddTest.java]: In next lessons we will learn new ways of writing tests that will allow us to write one method and test it against plenty of variations of data. But For now, when you are testing two different cases is better to split them

**Review by strzeleckik — CHANGES_REQUESTED** [review]
I left some comments with feedback

File: src/main/java/lesson2/task5/IsEvenOrOdd.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package lesson2.task5;
+
+public class IsEvenOrOdd {
+
+    public static boolean isEven(int number){
+        if (number %2 == 0) {
```
Comment (strzeleckik) [inline src/main/java/lesson2/task5/IsEvenOrOdd.java]: Take care of proper code formatting, ctrl + a and then ctrl + alt + l :)

**Comment by jean-farinea** [comment]
Changes done, please review.

**Review by strzeleckik — APPROVED** [review]
Good job! :)

**Pull request merged**